logdna/request_test.go

```diff
@@ -208,7 +209,8 @@ func TestRequest_MakeRequest(t *testing.T) {
 	t.Run("Handles errors when creating a new HTTP request", func(t *testing.T) {
 		const ERROR = "FAKE ERROR for body reader"
 		ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			json.NewEncoder(w).Encode(viewResponse{ViewID: "test123456"})
+			err := json.NewEncoder(w).Encode(viewResponse{ViewID: "test123456"})
+			assert.Nil(err, "No errors")
```

Comment:
Whoops, nice catch.  The new linter?

Reply:
yeah just a warning from the new lint suite's static checker